src/main/java/frc/robot/controls/BaseDriverConfig.java

```diff
@@ -57,15 +57,15 @@ public BaseDriverConfig(Drivetrain drive, ShuffleboardTab controllerTab, boolean
   }
 
   public double getForwardTranslation() {
-    return Functions.expoMS(MathUtil.applyDeadband(getRawForwardTranslation(), OIConstants.kDeadband), 2) * DriveConstants.kMaxSpeed * m_translationalSensitivity;
+    return -Functions.expoMS(MathUtil.applyDeadband(getRawForwardTranslation(), OIConstants.kDeadband), 2) * DriveConstants.kMaxSpeed * m_translationalSensitivity;
```

Comment:
Here and below: is reversion of driver controls needed for this PR?

Reply:
yeah, I was planning to remove that